suite-native/module-accounts-import/src/screens/AccountImportLoadingScreen.tsx

```diff
@@ -93,7 +110,7 @@ export const AccountImportLoadingScreen = ({
         return () => {
             ignore = true;
         };
-    }, [xpubAddress, networkSymbol, navigation]);
+    }, [xpubAddress, networkSymbol, navigation, dispatch]);
```

Comment:
nit: I think `dispatch` has guaranteed stable signature so it doesn't need to be in deps and it's also reason why eslint doesn't complain.